crates/services/operatorsinfo/src/fake_operator_info.rs

```diff
@@ -28,4 +30,11 @@ impl OperatorInfoService for FakeOperatorInfoService {
     ) -> Result<Option<OperatorPubKeys>, OperatorInfoServiceError> {
         Ok(self.pubkeys.pub_keys.clone())
     }
+
+    async fn get_operator_socket(
+        &self,
+        _address: Address,
+    ) -> Result<Option<String>, OperatorInfoServiceError> {
+        Ok(Some(self.socket.clone()))
+    }
```

Comment:
why is this function returning a Result, being that it's always returning Ok?

Reply:
the responder gives a Result with error okio::sync::oneshot::error::RecvError
the error should have been recv error instead of channel error 
```
 Ok(responder_rx
            .await
            .map_err(|_| OperatorInfoServiceError::ChannelClosed)?)
```